airflow/www/.prettierignore

```diff
@@ -0,0 +1,5 @@
+.mypy_cache/
+templates/**/*.html
+dist/
+*.md
```

Comment:
Don't prettier format yaml and md. Those are handled by other dedicated pre-commit hook.